### ISSUE TITLE
Motivating examples

### ISSUE THREAD

**Issue by sstevens2** [carpentries-incubator/jekyll-pages-novice#127]
@annefou @tobyhodges and I were discussing how the end product is not a beautiful website and how that might not be very motivating for developing a website using jekyll/gh-pages.  We thought it might be a good idea to show them one or more example webpages at the beginning and the repos that generate them.  This also then brings the idea that the repo makes the website up a bit sooner.

We would also say that the website we make we will build up piece by piece and while simpler than the examples it uses the same principles. Then when you (the learners) go to make your own website later you will be able to make use of a template that is more complicated because you are familiar with the  foundational pieces.

We could then weave this throughout the lesson by featuring one example site that then we come back to after each episode and show the repo behind again and point out the `index.md`, `_config.yml`, `_includes`, `_layouts`, etc.

Would really like to hear from @JulianKarlBauer @anenadic and @unode about this idea too.

Idea from debriefing on the [Nov 11 pilot](https://codimd.carpentries.org/KyHEeesgS7mXKjwwvmHcrw).

**Comment by unode** [comment]
At some point we discussed enabling a GitHub's theme. We tested selecting one after some commits have been added and that worked.
We could perhaps enable a theme at the end of the lesson, just as we mention pre-existing themes.
This brings some level of eye candy and maybe raises the satisfaction level.

We could also show the final screenshot in the introduction as a motivator  :star_struck:  for what is about to come.

For convenience, previews of github's themes:

- [Architect](https://pages-themes.github.io/architect/)
- [Cayman](https://pages-themes.github.io/cayman/)
- [Dinky](https://pages-themes.github.io/dinky/)
- [Hacker](https://pages-themes.github.io/hacker/)
- [Leap-Day](https://pages-themes.github.io/leap-day/)
- [Merlot](https://pages-themes.github.io/merlot/)
- [Midnight](https://pages-themes.github.io/midnight/)
- [Minima](https://jekyll.github.io/minima/)
- [Minimal](https://pages-themes.github.io/minimal/)
- [Modernist](https://pages-themes.github.io/modernist/)
- [Slate](https://pages-themes.github.io/slate/)
- [Tactile](https://pages-themes.github.io/tactile/)
- [Time-Machine](https://pages-themes.github.io/time-machine/)

**Comment by annefou** [comment]
@unode Yes I like the idea of finishing the lesson using a Github's theme but I think we said that it won't add all the necessary files in your repository (I haven't tried). What would be nice is to select a simple theme that would add all the config, layout files in your repo. Then we could go through as a final work along exercise.

**Comment by unode** [comment]
@annefou yes, you are right, it doesn't add files but since it's a built-in theme only the line in the config file is needed.

See the commit
https://github.com/unode/jekyll-sandbox/commit/e284bd52a6ecad6aebd9b7401a617f202d961d98
that was generated by GitHub after selecting the theme in the repo settings.
The rendered output at:
https://unode.github.io/jekyll-sandbox/
has the theme enabled.

When I tried it the first time, [Github replaced the simple `README.md` I had with some content](https://github.com/unode/jekyll-sandbox/commit/99d01ad). Didn't yet test if this happens when the README is mostly empty or if that's default behaviour for the first time a theme is selected for a repo.

Can anyone confirm if selecting a built-in GitHub theme at the end of the layout lesson works as expected?

**Comment by tobyhodges** [comment]
@unode my main concern with taking an approach like this is that, when we define our own page layouts later in the lesson, the connection to the theme will be broken. I may be wrong, but would like to check before we start changing things. Please could you try defining a layout (`_layouts/default.html`) in your `jekyll-sandbox` repo, apply it to a page, and report back on whether the theme styling was retained?

**Comment by unode** [comment]
@tobyhodges that's correct. Adding a `_layouts/default.html` overrides the GitHub theme.

**Comment by anenadic** [comment]
I agree that we should be aiming to weave in a real website somehow into the lesson. Maybe then showing a finished 'Capstone' screenshot of file structure of a repo would nicely fit with this. Perhaps something to think about for the next big iteration.

**Comment by tobyhodges** [comment]
Assigning this one to @anenadic as a follow-up to her work on #78

**Comment by tobyhodges** [comment]
At the March development sprint, it would be good to spend some time discussing what needs to be done to close this issue.

**Comment by sstevens2** [comment]
I think this is covered by the extras and the changes to the wrap-up section.  Closing but feel free to reopen or file a new issue if you think there is more to be addressed.
